code/modules/media/jukebox.dm

```diff
@@ -1041,6 +1066,11 @@ var/global/list/loopModeNames=list(
 		"holy" = "Pastor's Paradise"
 	)
 
+/obj/machinery/media/jukebox/holyjuke/update_icon()
+	..()
+	if(!(stat & (FORCEDISABLE|NOPOWER|BROKEN)) && anchored || !any_power_cut())
```

Comment:
Line 945 has "anchored **&&** !any_power_cut()" whereas you have an OR here - is that intentional?

Reply:
definitely not, thanks for catching that. I had fixed it for the superjuke but forgot to also change it here.